sites/fast-website/src/app/components/feature-card/feature-card.styles.ts

```diff
@@ -0,0 +1,67 @@
+import { css } from "@microsoft/fast-element";
+import { display, neutralForegroundRestBehavior } from "@microsoft/fast-components";
+
+export const FeatureCardStyles = css`
+    ${display("grid")} :host {
+        contain: layout;
+        grid-template-columns: 1fr 2fr;
+        color: inherit;
+        font-family: var(--text-font);
```

Comment:
A few of the other components use  var(--body-font). I was wondering if there was a benefit in using --text-font instead or if this could be changed to --body-font for consistency since they all have the same font-family in the figma.

Reply:
I just removed it for now until we have a defined one.